libpromises/processes_select.c

```diff
@@ -56,8 +56,13 @@ TABLE_STORAGE Item *PROCESSTABLE = NULL;
 
 #if defined(__sun) || defined(TEST_UNIT_TEST)
 static StringMap *UCB_PS_MAP = NULL;
-#define UCB_PS "/usr/ucb/ps"
-#define UCB_PS_ARGS "axwww"
+// These will be tried in order.
+const char *SOLARIS_UCB_STYLE_PS[] = {
+    "/usr/ucb/ps",
+    "/bin/ps",
```

Comment:
I'm a bit confused now. So on Solaris 11 both commands accept BSD-style arguments? And if `/usr/ucb/ps` is not available, we'll call `/bin/ps` once with System V arguments and once with BSD-style arguments?

What if UCB commands are missing from an older system, and `/bin/ps` barfs at the BSD-style arguments? Do we detect that it returns error?